packages/react/src/components/TextField/index.css

```diff
@@ -0,0 +1,87 @@
+.charcoal-text-field-root {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 4px;
+}
+
+.charcoal-text-field-root[aria-disabled='true'] {
+  opacity: 0.32;
+}
+
+.charcoal-text-field-container {
```

Comment:
Q: Just a minor question, why did you remove the gap and put the margin on the prefix, suffix side?

If there is a reason and I should copy it, I thought it would be good to know the reason for my decision.

Reply:
In Safari 13 and some other browsers, flexbox-gap is not supported.
https://caniuse.com/flexbox-gap

I made changes to broaden the support range.